ci/scripts/common.sh

```diff
@@ -105,7 +105,7 @@ function download_and_prepare_rw() {
   echo -e "\033[33mPrepare RiseDev dev cluster\033[0m"
 
   risedev pre-start-dev
-  risedev --allow-private link-all-in-one-binaries
+  risedev link-all-in-one-binaries
```

Comment:
Will this work?

Reply:
This is to avoid a warning complaining duplicated `--allow-private`